src/sanitizers.ts

```diff
@@ -0,0 +1,22 @@
+import { RemoveIndex } from './utils';
+import { ResponseType, UnwrapResponseType } from './responseType';
+
+import { ClientConfiguration } from '.';
+
+export type DataSanitizer<T> = (data: Partial<T>) => T;
+
+export type Sanitizers<Configuration extends ClientConfiguration> = {
+  [path in keyof RemoveIndex<Configuration>]?: {
+    [method in keyof Configuration[path]]?: Configuration[path][method] extends (
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      ...args: any
+    ) => unknown
```

Comment:
would it be possible to extract this to a type for better readability:
type Function = (...args: any) => unknown
Then I think it reads easier as a ternary